src/backends/sentry_backend_breakpad.cpp

```diff
@@ -23,10 +22,19 @@ extern "C" {
 
 #ifdef SENTRY_PLATFORM_WINDOWS
 #    include "client/windows/handler/exception_handler.h"
+#elif defined SENTRY_PLATFORM_DARWIN
```

Comment:
```suggestion
#elif defined(SENTRY_PLATFORM_DARWIN)
```

Can you change these to be consistent with the rest of the codebase please.